src/eth/block.yaml

```diff
@@ -11,9 +11,14 @@
         title: hydrated
         type: boolean
   result:
-    name: Block information
+    name: Block Result
     schema:
-      $ref: '#/components/schemas/Block'
+      oneOf:
+        - title: Not Found (null)
```

Comment:
I think it would be better to define a null type with this title and `$ref` it, so that the schema looks like

```
oneOf:
  - $ref: '#/components/schemas/Block'
  - $ref: '#/components/schemas/null'
  ```

Reply:
or maybe `'#/components/schemas/NotFound'`

Reply:
Why though, `type: 'null'` is shorter.

Reply:
I believe it should just pull in the titles from the underlying schema, no need to define again. Need to double check this though.

Reply:
Ahh, wait, you mean because of the title.

Reply:
Yeah it doesn't seem great to duplicate the title?

Reply:
I think we need to 1) remove the title for the block data, because it simply overrides the title in the block object schema. We can just rely on that via the ref. And 2) define a null type for not found, so we don't need to duplicate the title across uses.